src/Compiler/TypedTree/TypedTreeOps.fs

```diff
@@ -9573,11 +9573,11 @@ type Entity with
                 List.lengthsEqAndForall2 (typeEquiv g) (List.map fst argInfos) argTys &&  
                 membInfo.MemberFlags.IsOverrideOrExplicitImpl
             | _ -> false) 
-    
-    member tycon.HasMember g nm argTys = 
+
+    member internal tycon.TryGetMember g nm argTys = 
```

Comment:
Everything in TypedTreeOps is internal, and in any case no need to make that explicit when using signature files.